src/Components/Alert/Alert_Controller.php

```diff
@@ -27,10 +27,21 @@ public function render(): void {
 
 		echo $this->view->render( self::VIEW, [
 			'dto'             => $alert,
+			'alert_classes'   => $this->get_alert_classes( $alert ),
```

Comment:
```suggestion
			'classes'   => $this->get_alert_classes( $alert ),
```
Let's just call this classes

Reply:
Renamed.